Docker/start-container.sh

```diff
@@ -6,11 +6,11 @@ PUID=${PUID:-`id -u squeezeboxserver`}
 PGID=${PGID:-`id -g squeezeboxserver`}
 
 usermod -o -u "$PUID" squeezeboxserver
-groupmod -o -g "$PGID" squeezeboxserver
+groupmod -o -g "$PGID" nogroup
 
 umask 0002
 
 #Add permissions
-chown -R squeezeboxserver:users /config /playlist
+chown -R squeezeboxserver:nogroup /config /playlist /lms
```

Comment:
Why does squeezeboxserver need ownership of /lms? LMS only writes files in /config.